src/github_changelog/conventional.clj

```diff
@@ -110,8 +108,7 @@
       (conj pulls pull))))
 
 (defn ^:no-gen parse-changes [config {:keys [pulls] :as tag}]
-  (->> (map (partial parse-pull config) pulls)
-       (remove nil?)
+  (->> (keep (partial parse-pull config) pulls)
        (reduce filter-reverted [])
```

Comment:
If you rewrite `filter-reverted` to be pred-ish (sorta like `(defn reverted? [pulls pull] ,,,)`), you could write these 2 lines as `(into [] (comp (keep (partial ,,,)) (remove reverted?)) pulls)`, I think.

Reply:
Could also comp in the next line then

Reply:
I also though that but `filter-reverted` needs to see the full `keep` produced seq. Therefore you can't use on it on a single element inside a transducer chain.

Reply:
True, I didn't realise `pulls` is that seq.